genome_sampler/plugin_setup.py

```diff
@@ -426,6 +426,40 @@ def _5(fmt: VCFLikeMaskFormat) -> pd.DataFrame:
                 ' that will be removed from all sequence ids.'
 )
 
+plugin.methods.register_function(
+    function=label_unaligned_seqs,
+    inputs={'seqs': FeatureData[Sequence]},
+    parameters={
+        'delimiter': Str % Choices('|', ',', '+', ':', ';'),
+        'metadata': Metadata,
+        'columns': List[Str],
+        'missing_value': Str,
+    },
+    outputs=[('labeled_seqs', FeatureData[Sequence])],
+    input_descriptions={'seqs': 'The sequences to be re-labeled.'},
+    parameter_descriptions={
+        'delimiter': 'The delimiter between the sequence id and each metadata'
+                     ' entry.',
+        'metadata': 'The metadata to embed in the header.',
+        'columns': 'The columns in the metadata to be used.',
+        'missing_value': 'Value to use to indicate missing metadata column '
+                         'values for sequences.'
+    },
+    output_descriptions={
+        'labeled_seqs': 'The re-labeled sequences.'
+    },
+    name='Re-label unaligned sequences (deprecated)',
```

Comment:
Maybe you should include a `deprecated=True` argument in this method registration, too, that way interfaces will indicate the deprecated status.

Reply:
I can put a PR in for that, too - just let me know.

Reply:
Please see #61.